Cubical/Functions/Involution.agda

```diff
@@ -0,0 +1,43 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Functions.Involution where
+
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Univalence
+
+isInvolution : ∀{ℓ} {A : Type ℓ} → (A → A) → Type _
+isInvolution f = ∀ x → f (f x) ≡ x
+
+module _ {ℓ} {A : Type ℓ} {f : A → A} (invol : isInvolution f) where
+  open Iso
+
+  involIso : Iso A A
+  involIso .fun = f
+  involIso .inv = f
+  involIso .rightInv = invol
+  involIso .leftInv = invol
+
+  involIsEquiv : isEquiv f
+  involIsEquiv = isoToIsEquiv involIso
```

Comment:
Hmm, isn't there a more direct proof of this which should be simpler than what you get if you unfold `isoToIsEquiv`? I would guess that you can pretty much directly write down the `isEquiv` proof

Reply:
I can try, but personally I find `isEquiv` to be very difficult to prove directly in almost every situation.

Reply:
I agree. Filling the square takes some thinking and the feedback one gets from Agda isn't too helpful.

Reply:
I tried to do it myself, but can't find a simple way... So let's leave it as it is